app/sagas/createChannel.js

```diff
@@ -49,14 +71,22 @@ const handleRequest = function* handleRequest({ data }) {
 				encrypted
 			});
 			sub = yield call(createChannel, data);
-		}
 
+			if (data.teamId) {
```

Comment:
Front end started using `groups.create` and `channels.create` for this.
They added a new param to the api to send a `teamId`, so this extra step isn't needed.
![image](https://user-images.githubusercontent.com/804994/117307698-29b44580-ae57-11eb-9501-62b19e30919e.png)